src/model/structures/GameState.cs

```diff
@@ -6,21 +6,21 @@
 
 namespace tcg
 {
-    struct GameState
-    {
-        public bool IsFinished {get; set;}
-        public Player CurrentPlayer {get; set;}
-        public Player[] Players {get; set;}
-        public int Target {get; set;}
-        public int Attacker {get; set;}
+  class GameState
+  {
+    public bool IsFinished { get; set; }
+    public Player CurrentPlayer { get; set; }
+    public Player[] Players { get; set; }
+    public int Target { get; set; }
+    public int Attacker { get; set; }
```

Comment:
I even don't know..
Do we actually need them to be properties?
This CamelCase for field looks.. strange?
But it's not critical anyway